### PR TITLE
javadoc: add Chinese translation

### PR DESCRIPTION
**Pull request by BillLucky**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/common/javadoc.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- 生成 Java 源代码的文档并将结果保存在文件夹中：
+
+`javadoc -d {{path/to/directory/}} {{path/to/java_source_code}}`
+
+- 生成指定编码的文档：
+
+`javadoc -docencoding {{UTF-8}} {{path/to/java_source_code}}`
+
+- 生成文档（不包括某些软件包）：
```
Comment (marchersimon) [inline pages.zh/common/javadoc.md:14]: I don't think the `excluding some packages` should go in parentheses here.

Comment (BillLucky) [reply pages.zh/common/javadoc.md]: Okay, I have fixed it and make it clearer @marchersimon

**Review by marchersimon — APPROVED** [review]
Thanks @BillLucky

**Review by bl-ue — APPROVED** [review]
Cool, thank you @BillLucky.  Really cleanly cut ;)

**Pull request merged**